apps/backend/src/controllers/property.controller.ts

```
@@ -304,6 +348,12 @@ export async function updatePropertyController(
         updateData.property_token === undefined
           ? null
           : updateData.property_token,
+      availability: Array.isArray(updateData.availability)
+        ? updateData.availability.map((range: any) => ({
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)